content/dependencies.md

```diff
@@ -25,7 +25,7 @@ From [xkcd - dependency](https://xkcd.com/2347/). Another image that might be fa
 ````{discussion} Kitchen analogy
 - Software <-> recipe
 - Data <-> ingredients
-- Libraries <-> cooking books/blogs
+- Libraries <-> Pots/Tools
```

Comment:
Sounds sensible to me. 

One thing about consistency: uppercase or lowercase? Maybe all uppercase makes sense here, then above analogies should also be adapted.